api/controllers/CheckUploadedDocumentsController.js

```diff
@@ -14,20 +17,30 @@ const CheckUploadedDocumentsController = {
             if (uploadedFileData.length === 0) {
                 throw new Error('No files uploaded');
             }
-            uploadedFileData.forEach(async (uploadedFile) => {
-                await UploadedDocumentUrls.create(
+            uploadedFileData.forEach((uploadedFile) => {
+                UploadedDocumentUrls.create(
                     CheckUploadedDocumentsController._dbColumnData(
                         uploadedFile,
                         req
                     )
-                );
-                sails.log.info(
-                    `Url for document ${uploadedFile.filename} added to db`
-                );
+                )
+                    .then(() => {
+                        sails.log.info(
+                            `Url for document ${uploadedFile.filename} added to db`
+                        );
+                    })
+                    .catch((err) => {
+                        sails.log.error(err);
+                        res.serverError();
+                    });
             });
-            res.redirect('/check-uploaded-documents');
+
+            CheckUploadedDocumentsController._checkDocumentCountAndPaymentDetails(
+                req,
+                res
+            );
         } catch (err) {
-            sails.log.error(err);
+            sails.log.error(err.message, err.stack);
```

Comment:
Any reason this is different to above? `sails.log.error(err);`?

Reply:
So the one above prints out the whole error stack, but in one of the unit tests I wanted to test just the message and not the whole stack, so I amended it to split the message and the stack so I can at least test the message is printed.

Reply:
Ah okay, I'll give that a go

- [ ] Change error type here to trace or debug

Reply:
Hmm, when I first read your comment I thought you were referring to the sails logging options. https://sailsjs.com/documentation/concepts/logging 

But it doesn't have a `trace`. Which has got me thinking maybe you were referring to something else 🤔

Reply:
you can go for `debug` or `info`. A lot of loggers will provide `trace` so that's where I got it from. trace is usually more verbose than debug